frontend/app/routes-flow/apply-flow.ts

```diff
@@ -81,6 +81,13 @@ const communicationPreferencesStateSchema = z.object({
   preferredLanguage: z.string().min(1),
 });
 
+/**
+ * Schema for demographic reference.
+ */
+const demographicsPart1StateSchema = z.object({
+  otherEquity: z.string().min(1).optional(),
```

Comment:
```suggestion
  otherEquity: z.string().trim().min(1).optional(),
```

to prevent all-whitespace inputs